docs/docs/deployment/hosting/kubernetes.md

```diff
@@ -478,7 +478,7 @@ The following table lists the configurable parameters of the chart and their def
 
 ## Key upgrade notes
 
-- [0.20.0](https://artifacthub.io/packages/helm/flagsmith/flagsmith/0.20.0): upgrades the bundled in-cluster Postgres.
+- [0.37.0](https://artifacthub.io/packages/helm/flagsmith/flagsmith/0.37.0): upgrades the bundled in-cluster Postgres.
   This makes no effort to preserve data in the bundled in-cluster Postgres if it is in use. This also renames the
   bundled in-cluster Postgres to have `dev-postgresql` in the name, to signify that it exists such that the chart can be
```

Comment:
This is different to what we have above? Is it `dev-postgresql` or `devPostgresql`?

Reply:
That refers to the name of the pod, i.e: the pod that is running the database will have `dev-postgresql` in the name. `devPostgresql` on the other hand is value/config

Reply:
Ok 👍